landlab/components/soil_grading/soil_grading.py

```diff
@@ -209,14 +208,15 @@ def __init__(
         self._initial_median_size = initial_median_size
         self._CV = CV
         self._initial_total_soil_weight = initial_total_soil_weight
-        if std == None:
+        if std is None:
             std = self._initial_median_size * self._CV
         self._std = std
         self._is_bedrock_distribution_flag = is_bedrock_distribution_flag
         self._precent_of_volume_in_spread = precent_of_volume_in_spread
 
         # Create out fields.
-        # Note: Landlabs' init_out_field procedure will not work for the 'grains__weight' and 'grains_fractions__size' fields
+        # Note: Landlabs' init_out_field procedure will not work
+        # for the 'grains__weight' and 'grains_fractions__size' fields
```

Comment:
Also, I'm wondering whether the try...except around soil__depth is necessary. Seems like this forces there not to be a pre-existing soil__depth field, yet I can imagine some applications wanting to start off with a given soil__depth field.

Reply:
Thanks. Yes, it forces that soil__depth field will not be created outside of SoilGrading. The point is-- if a soil__depth field is created outside of SoilGrading, the grid is not associated with the corresponding grain size information. The idea is not to allow this to happen in case SoilGrading is in use. Note that if a user wants to start with a given soil__depth field, SoilGrading can be initialized with a desired grain size distribution and total soil weight and it will create a corresponding soil__depth field.